### PR TITLE
Уткин Константин - Лабораторная работа #3

### PR DESCRIPTION
**Pull request by megarungle**

### CODE REVIEW

**Comment by ScautX** [comment]
:+1:

**Comment by megarungle** [comment]
@ScautX, @alinaut

**Comment by alinaut** [comment]
:+1:

**Comment by megarungle** [comment]
@alexbatashev, @TheG1uy, @Agricola87, review pls

File: modules/queue/src/queue_app.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Utkin Konstantin
+
+#include <string>
+#include <cstring>
+#include <cstdlib>
+#include <iostream>
```
Comment (gooddoog) [inline modules/queue/src/queue_app.cpp]: Неиспользуемая библиотека iostream

File: modules/queue/include/queue_app.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#ifndef MODULES_QUEUE_INCLUDE_QUEUE_APP_H_
+#define MODULES_QUEUE_INCLUDE_QUEUE_APP_H_
+
+#include <string>
+#include "include/queue.h"
+
+class QueueApplication {
+ private:
+    std::string message_;
+    Queue que;
```
Comment (gooddoog) [inline modules/queue/include/queue_app.h]: Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a **trailing underscore**.
https://google.github.io/styleguide/cppguide#Variable_Names

File: modules/queue/src/queue_app.cpp (hunk 1 above)
Comment (megarungle) [reply modules/queue/src/queue_app.cpp]: Удалено.

File: modules/queue/include/queue_app.h (hunk 2 above)
Comment (megarungle) [reply modules/queue/include/queue_app.h]: Thank you, fixed.

**Comment by gooddoog** [comment]
:+1:

**Pull request merged**